src/index.js

```diff
@@ -11,7 +11,7 @@ let CloseButton = (props) => (props)
 let goBack = () => ('/')
 
 let LegendedButton = ({children, title}) => (
-    <div style={{display: 'flex', flexDirection: 'column'}}>
+    <div style={{display: 'flex', flexDirection: 'row'}}>
```

Comment:
@xaiki why are we using inline styles?

Reply:
because we're waiting on someone that «actually knows what he's doing» to come and fix it =)